### ISSUE TITLE
Are there plans to have this on quelpa ?

### ISSUE THREAD

**Issue by 573** [rougier/nano-emacs#19]
Hi,

I stumbled over your elegant-emacs package which I found awesome but somehow [the custom install instructions](https://github.com/rougier/elegant-emacs/issues/4#issuecomment-735210190) end with the link here.

Now I would love to install nano-emacs via quelpa as was suggested for elegant-emacs.

**Comment by rougier** [comment]
Yes, we could try to have the different modules on quelpa. I think it's only a matter of adding a quelpa recipe. Can you make a PR?

**Comment by 573** [comment]
I have no experience with the process so if someone else is willing to do this [this](https://github.com/melpa/melpa/pull/7134) might be a good template.

**Comment by dixius99** [comment]
Once you have Quelpa installed, can't you just add the following to your `init` file?

```
(quelpa '(nano-emacs :repo "rougier/nano-emacs" :fetcher github))
```

I.e. Quelpa can pull the repo directly from GitHub, can't it? I don't use Quelpa, but I know [`straight.el`](https://github.com/raxod502/straight.el) does this (which is what I use). The formula for `straight` is:

```
(straight-use-package
  '(nano-emacs :type git :host github :repo "rougier/nano-emacs"))
```
